src/api/spec/models/package_spec.rb

```diff
@@ -398,10 +398,11 @@
     end
   end
 
-  describe '#jobhistory_list' do
+  describe '::jobhistory_list' do
```

Comment:
[Nitpick] everywhere else we use ``.`` instead of ``::`` for class methods.

Reply:
why do we use `.`? :confused:

Reply:
I would prefer `::` as well since that's what is usually used class methods. But one way or the other we should align this to one way of writing this.

Reply:
Here you go: https://github.com/openSUSE/open-build-service/pull/5775 :wink: